src/components/Promo/PromoCard.tsx

```diff
@@ -0,0 +1,101 @@
+import { Button } from '@chakra-ui/react'
+import dayjs from 'dayjs'
+import isBetween from 'dayjs/plugin/isBetween'
+import { useCallback, useMemo } from 'react'
+import { useHistory } from 'react-router-dom'
+import FoxyPromo from 'assets/foxy-promo.jpg'
+import { Card } from 'components/Card/Card'
+import { Carousel } from 'components/Carousel/Carousel'
+import { RawText } from 'components/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+dayjs.extend(isBetween)
+
+const promoData = [
+  {
+    title: 'Earn 3.15% APY on FOX',
+    body: 'Your FOX is put to work across different DeFi protocols to earn the best yield possible.',
+    cta: 'Deposit FOX Now',
```

Comment:
Translations?

Reply:
These aren't going to be translated --- the idea was that marketing can go in and add these pretty easily via that array of promos. Not sure if we want to have to translate this one off cards.